src/main/java/edu/suffolk/litlab/efspserver/ecf4/TylerModuleSetup.java

```diff
@@ -198,8 +200,15 @@ public void preSetup() {
     log.info("Done checking table if absent");
 
     try {
+      boolean disableQuartzSchedule =
+          Boolean.parseBoolean(GetEnv("DISABLE_QUARTZ_SCHEDULE_FOR_CODE_UPDATE").orElse("false"));
```

Comment:
Sounds good! I'll update along w/ the documentation.

Looking at this again, I don't think it's worth it to even execute the rest of the logic if the schedule is disabled, since there isn't a mechanism to re-enable the schedule without a restart. Will probably change this to short-circuit return from the method call.